lib/ansible/modules/cloud/openstack/os_volume.py

```diff
@@ -46,6 +46,10 @@
      description:
        - Volume name or id to create from
      version_added: "2.3"
+   bootable:
+     description:
+       - Bootable flag for volume
+     version_added: "2.7"
```

Comment:
Can you set this to 2.8?  Sorry for the delay in how long this took.  We'll get this merged soon.

Reply:
sure